app/models/host/base.rb

```diff
@@ -400,7 +401,7 @@ def get_interface_scope(name, attributes, base = self.interfaces)
         # we search bonds based on identifiers, e.g. ubuntu sets random MAC after each reboot se we can't
         # rely on mac
         when 'Nic::Bond', 'Nic::Bridge'
-          base.virtual.where(:identifier => name)
+          base.where(:identifier => name)
```

Comment:
This allows to update physical interface to bond/bridge, which is virtual. Reading #23654 it seems that you defined the br_private interface defined already and not marked as bridge. This does not seem to be a bug but rather a misconfiguration. Or could you explain pls, why we need to search also physical interfaces?

Reply:
@ares the issue with primary bridge interface because in this case the information already in the DB with incorrect :virual set to false. BTW the :identifier is unique so my proposed changes should work in all cases.